modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase

```diff
@@ -632,7 +632,7 @@ definition {
 
 		Audit.viewAuditEntries(
 			clientIP = "127.0.0.1",
-			createDate = "Seconds Ago",
+			createDate = "Second Ago",
 			id = "",
```

Comment:
From the test history, this is failing intermittently. The problem is that if the test is too fast, it'll navigate to the page in one second. If we apply the fix, it'll fail every time it takes longer than one second to nagivate to the page.

We should find a way for this to work 100% of the time. For example, you could try is adding a refresh after Audit.openAuditAdmin(); if it's the table contains "1 Second Ago".

Reply:
@andreefarias please find a method to make the test more resilient. You can try using the example above or find another method.